config/core/deployments/broker.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: broker-ingress
+  namespace: cloud-run-events
+  labels:
+    events.cloud.google.com/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: cloud-run-events
+      role: broker-ingress
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+      - name: broker-ingress
+        image: github.com/google/knative-gcp/cmd/broker/ingress
+        env:
+          - name: PORT
+            value: "8080"
+        volumeMounts:
+        - mountPath: /var/run/cloud-run-events/broker
+          name: broker-config
+        resources: # TODO(issue #876): determine good values for resource requests/limits
```

Comment:
Do we need parameters for metrics/tracing/logging?

Reply:
No, not at the moment. I heard that eventing may stop supporting the configmap feature for metrics/tracing/logging. Need to follow up on that.

Reply:
Hmm. What's the replacement? Operator?

Reply:
I didn't follow up on that. I heart from @grantr .

Maybe just stop supporting dynamic config.

Reply:
This was a conversation related to the dissolution of the Knative Observability working group. Evan questioned if the complexity of the dynamic configuration was worth maintaining. No concrete guidance yet.